mercury.spec

```diff
@@ -21,6 +21,7 @@ URL:		http://mercury-hpc.github.io/documentation/
 Source0:	https://github.com/mercury-hpc/%{name}/archive/v%{version}.tar.gz
 Patch1:		https://github.com/mercury-hpc/mercury/compare/v2.0.0a1..%{mercury_commit}.patch
```

Comment:
does this have any issues?

Reply:
Not sure. I believe is patch 2 which Yulu is rolling back.